README.md

```diff
@@ -213,6 +213,12 @@ Each `llm` component is defined by two main settings:
 Moreover, the component also implements [**caching**](#cache) functionality to avoid running
 the same document through an LLM service (be it local or through a REST API) more than once.
 
+Finally, you can choose to save a stringified version of LLM prompts/responses
```

Comment:
I think the README could use a table of `llm` config settings?

Reply:
I agree, it's overdue. I added a table and kept the two main settings apart like before, it seems important. Let me know what you think.